geowave-benchmark/src/main/java/mil/nga/giat/geowave/benchmark/utils/HistogramGenerator.java

```diff
@@ -354,7 +354,6 @@ public void originalTest(
 		final List<Double> featsPerTile10 = new ArrayList<Double>();
 		final Map<Integer, Integer> collsPerTier10 = new HashMap<Integer, Integer>();
 		final Map<Integer, Integer> featsPerTier10 = new HashMap<Integer, Integer>();
-		final int[] hist10 = new int[500];
```

Comment:
@jwomeara 
A few deletions like this in HistogramGenerator - take a looks, it looks like the values were never actually used (they were written two, but never read.